pa11ycrawler/cmdline.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python
+"""
+The command line interface for pa11ycrawler.
+"""
+import argparse
+
+from textwrap import dedent
+
+from scrapy.crawler import CrawlerProcess
+
+from pa11ycrawler.reporter import HtmlReporter
+from pa11ycrawler.settings_helpers import (
+    is_pa11y_setting,
+    is_pa11ycrawler_setting,
+    get_default_settings,
+)
+from pa11ycrawler.spiders.pa11y_spider import Pa11ySpider
+
+# ------------ Main CLI ------------
+
+
+def cli():
+    """
+    Returns the main CLI parser.
+    """
+    parser = argparse.ArgumentParser(prog='pa11ycrawler')
+    subparsers = parser.add_subparsers(title='commands')
+    run_cli(subparsers)
+    json_to_html_cli(subparsers)
+    return parser
+
+# ------------ "run" CLI -----------
+
+
+def run_cli(subparsers):
+    """
+    Creates the parser for the `run` command
+    """
+    run_usage = dedent('''pa11ycrawler run [options] url [url ...]
+
+    ---------------------------------------------------------------
+    NOTE: This only documents a haldful of the possible options.
+    More documentation can be found for the following categories:
+
+    CUSTOM SCRAPY SETTINGS:
+    ** Implemented for pa11ycrawler. (e.g. Not found in Scrapy Docs.)
+    ** All options documented below.
+    ** Most likely to need to be changed.
```

Comment:
Oh I understand. I think it was confusing to me as its own bullet-point. Maybe you could collapse the two bullets into one. Something like "All options documented below, and most likely need to be set when running pa11ycrawler". Something like that? 

Perhaps it's just 'changed'...you could also leave it as two bullets and make the 2nd bullet something like "Most settings will need to be set when running crawler"